wrappers/python/pyrealsense2.cpp

```diff
@@ -54,25 +54,73 @@ PYBIND11_MODULE(NAME, m) {
         .def("__str__", &rs2::log_message::raw)
         .def("__repr__", &rs2::log_message::full);
 
-    m.def("log_to_callback",
-        [](rs2_log_severity min_severity, std::function<void(rs2_log_severity, rs2::log_message)> callback)
+    // We want to enable Python callbacks for logging, but need to be careful:
+    // The machanism used by librealsense keeps a pointer to an object that is then released
+    // on destruction/exit. Usually this works fine, except that here, with Python and its GIL,
+    // Pybind tries to acquire the GIL when the thread state is no longer valid and we get
+    // into an infinite wait.
+#if 0
```

Comment:
Please explain the aim of keeping this #if 0

Reply:
I updated the comment as discussed